Makefile

```diff
@@ -188,9 +189,15 @@ publish: docker-build ## Push images on to Container Registry (default: ghcr.io)
 	docker push $(IMAGE_CONTROLLER)
 	docker push $(IMAGE_ADAPTER)
 
+publish-controller-multiarch:
+	docker buildx build --output=type=$(OUTPUT_TYPE) --platform=linux/amd64,linux/arm64 . -t ${IMAGE_CONTROLLER} --build-arg BUILD_VERSION=${VERSION} --build-arg GIT_VERSION=${GIT_VERSION} --build-arg GIT_COMMIT=${GIT_COMMIT}
+
+publish-adapter-multiarch:
+	docker buildx build --output=type=$(OUTPUT_TYPE) --platform=linux/amd64,linux/arm64 -f Dockerfile.adapter -t ${IMAGE_ADAPTER} . --build-arg BUILD_VERSION=${VERSION} --build-arg GIT_VERSION=${GIT_VERSION} --build-arg GIT_COMMIT=${GIT_COMMIT}
+
 publish-multiarch:
-	docker buildx build --push --platform=linux/amd64,linux/arm64 . -t ${IMAGE_CONTROLLER} --build-arg BUILD_VERSION=${VERSION} --build-arg GIT_VERSION=${GIT_VERSION} --build-arg GIT_COMMIT=${GIT_COMMIT}
-	docker buildx build --push --platform=linux/amd64,linux/arm64 -f Dockerfile.adapter -t ${IMAGE_ADAPTER} . --build-arg BUILD_VERSION=${VERSION} --build-arg GIT_VERSION=${GIT_VERSION} --build-arg GIT_COMMIT=${GIT_COMMIT}
+	publish-controller-multiarch
+	publish-adapter-multiarch
```

Comment:
These 3 lines should be the same line.
```
publish-multiarch: 	publish-controller-multiarch publish-adapter-multiarch
```
This is because in Makefile, the commands in the same line are other commands defined in the Makefile and commands below it, are shell commands

Reply:
I have tried and this works well, just applying the previous comment 😄 
Once it's done, we can merge it! 
Thanks a ton ❤️